pkg/operator/sync.go

```diff
@@ -556,6 +556,21 @@ func newPodTemplateSpec(config *OperatorConfig, features map[string]bool) *corev
 				},
 			},
 		},
+		{
+			Name: "bound-sa-token",
+			VolumeSource: corev1.VolumeSource{
+				Projected: &corev1.ProjectedVolumeSource{
+					Sources: []corev1.VolumeProjection{
+						{
+							ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+								Audience: "openshift",
```

Comment:
I don't understand why we need this.

Reply:
the token or the `openshift` audience?

We need the token to present to STS to do the AssumeRoleWIthWebIdentity calll (which will be validated by the OpenID Connect Provider registered in IAM)

We need to set the audience to the same audience the OpenID Connect Provider was registered to accept.  `openshift` is an arbitrary, but platform-neutral choice.

Reply:
Should this be setting the expiry? IIRC that's a field in this struct right? What happens if we don't set the expiry?

Reply:
It defaults to 1h, which is in alignment with STS guidelines
https://github.com/kubernetes/kubernetes/blob/4b59044b8d2a3502ea490ba2c958008a098511a3/pkg/apis/core/v1/defaults.go#L273

Which is in alignment with default STS AssumeRole
https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
> By default, the temporary security credentials created by AssumeRole last for one hour.

We can explicitly set it if you prefer.  Just let me know.

Reply:
I'd prefer to see it explicitly set with a comment saying this is the recommended default and linking to the AWS docs if that's ok?